addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -467,6 +433,24 @@
             </t>
         </t>
     </div>
+    <t t-call="website_slides.share_slide_modal">
+        <t t-set="record" t-value="slide"/>
+        <t t-set="include_embed" t-value="bool(slide.embed_code)"/>
+        <t t-set="share_slide_modal_title">Share This Document</t>
+        <t t-set="website_share_url" t-value="slide.website_share_url"/>
+    </t>
+</template>
+
+<!-- Share: social media -->
+<template id='share_slide_modal'>
```

Comment:
Looks like we already have a "slide_share_modal" template, we can't have a "share_slide_modal" that looks almost identical.

Can't you re-use the other one?